### PR TITLE
fix(storage): sync title length constraint with API

### PR DESCRIPTION
**Pull request by scop**
Ref https://developers.upcloud.com/1.3/9-storages/

### CODE REVIEW

**Comment by kangasta** [comment]
Could add CHANGELOG entry for this as well

**Review by villevsv-upcloud — APPROVED** [review]

**Comment by scop** [comment]
> Could add CHANGELOG entry for this as well

Cheers, done in 0fe478b83f3d155a588e072b2bcc4112448cccf9

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ See updating [Changelog example here](https://keepachangelog.com/en/1.0.0/)
 
 - upcloud_loadbalancer_frontend: handle changes in the `networks`
 - upcloud_loadbalancer: set `maintenance_dow` and `maintenance_time` as computed to avoid planning them to be removed when missing from configuration.
+- upcloud_storage: sync title length constraint with API, allows 1-255 characters now
```
Comment (kangasta) [inline CHANGELOG.md:24]: This is now under _5.8.0_ instead of _Unreleased_ 🕵️‍♂️

Comment (scop) [reply CHANGELOG.md]: Damn, I did not have a time machine after all. Force-fixed in 4b90c68c65588a439d9712aa680fe0b87e567e5b

**Pull request merged**